.gitignore

```diff
@@ -9,4 +9,7 @@ node/
 package-lock.json
 cypress/screenshots
 cypress/videos
-node_modules/
\ No newline at end of file
+node_modules/
+src/test/java/org/openmrs/contrib/qaframework/RunTest.java
+src/test/resources/org/openmrs/uitestframework/test.properties
```

Comment:
can you now try making changes to the files ignored here locally and run `git status` and see if they aren't tracked!

Reply:
> can you now try making changes to the files ignored here locally and run `git status` and see if they aren't tracked!

I realized a cache somewhere and I have figured out the best way to resolve this as addressed in the updated the pr

Reply:
In my approach to resolve this issue, Removing these two files from the cached repository (but not from the file system) achieves the intended goal!! Locally, when i do any change in any of either file and run git status, the result is what we wish committers to experience. cc: @k-joseph

Reply:
You just ended up deleting those 2 files

Reply:
> You just ended up deleting those 2 files

Am figuring out the best approach to this

Reply:
have you found a way forward?

Reply:
> have you found a way forward?

I figured out the way to go and tested it locally and working fine.